scio-repl/src/main/scala-2.12/com/spotify/scio/repl/compat/CsvReaderOps.scala

```diff
@@ -4,6 +4,6 @@ import kantan.csv.CsvReader
 
 final class CsvReaderOps[A](val reader: CsvReader[A]) extends AnyVal {
 
-  def iterator: Iterator[A] = reader.toIterator
+  def iterator: Iterator[A] = reader.iterator
```

Comment:
same here

Reply:
Seems to work for me locally will double check.

Reply:
Yeah this is fine. There was another breakage but all fixed.

Reply:
ahhh cool! 😄 sorry about this I confused this with the one provided by `kantan` 🤦